spec/01-unit/19-hybrid/03-compat_spec.lua

```diff
@@ -630,5 +630,110 @@ describe("kong.clustering.compat", function()
       assert.is_nil(assert(services[3]).ca_certificates)
     end)
 
-  end)
+  end)  -- describe
+
+  describe("route entities compatible changes", function()
+    _G.kong = { configuration = { router_flavor = "expressions" } }
```

Comment:
Should we use `lazy_setup` and `lazy_teardown` to set up and tear down the test environment?
If there are other tests following this one, it may affect the execution of those tests.